week2/utilities/ltr_utils.py

```diff
@@ -59,7 +77,8 @@ def create_feature_log_query(query, doc_ids, click_prior_query, featureset_name,
                         "featureset": "bbuy_main_featureset",
                         "store": ltr_store_name,
                         "params": {
-                            "keywords": query
+                            "keywords": query,
+                            "click_prior_query": click_prior_query
```

Comment:
@HuanfengMa thanks for spotting this. I had the commit on my local branch however I had not pushed it to github yet!